src/Tribe/Cache.php

```diff
@@ -13,6 +13,7 @@ class Tribe__Cache implements ArrayAccess {
 	const SCHEDULED_EVENT_DELETE_TRANSIENT = 'tribe_schedule_transient_purge';
 	const NO_EXPIRATION  = 0;
 	const NON_PERSISTENT = - 1;
+	const PERSISTENT_GROUP = 'tribe-events';
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Equals sign not aligned with surrounding assignments; expected 17 spaces but found 1 space

<sub>hash: fbc6638102e7ca25ff4df7a986949812</sub>